src/meta/meta_ckpt.c

```diff
@@ -1262,6 +1262,195 @@ __wt_meta_ckptlist_update_config(
     return (ret);
 }
 
+/*
+ * __ckpt_extract_blkmod_bitmap --
+ *     Extracts the blkmod info in a checkpoint into a bitmap in a WT_ITEM.
```

Comment:
The sentence looks odd to me: in... into... in.
```suggestion
 *     Extracts the blkmod info in a checkpoint into a bitmap.
```
If we really want to mention `WT_ITEM`, I'd suggest:
```suggestion
 *     Extracts the blkmod info in a checkpoint into a WT_ITEM.
```

Reply:
I've changed it to "Extracts the blkmod info in a checkpoint into a bitmap within a WT_ITEM." How does that sound?